website/gitlab.erb

```diff
@@ -46,6 +46,9 @@
           <li<%= sidebar_current("docs-gitlab-resource-user") %>>
             <a href="/docs/providers/gitlab/r/user.html">gitlab_user</a>
           </li>
+          <li<%= sidebar_current("docs-gitlab-resource-service_slack") %>>
```

Comment:
What about the Jira one?

Reply:
It's not done yet. The Jira integration is missing doc and tests. I'm waiting for `go-gitlab` vendor release before doing a proper PR which will include everything and will target claranet_rebased_upstream branch.